### PR TITLE
[LineSection] Clean useless properties in files

### PR DESCRIPTION
**Pull request by tchevily**
# Description

This PR removes unused attribute ''sens'' of line_section

## Issue (optional)

Issue link: BOT-1914

## Pull Request type (optional)

This is a technical debt

### CODE REVIEW

**Comment by tchevily** [comment]
Jenkins can you test this please?

**Comment by tchevily** [comment]
Jenkins can you test this please?

File: chaos/default_settings.py [hunk 1]
```diff
@@ -2,46 +2,46 @@
 # URI for postgresql
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 # http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-SQLALCHEMY_DATABASE_URI = str(os.getenv('SQLALCHEMY_DATABASE_URI', 'postgresql://navitia:navitia@localhost/chaos'))
+SQLALCHEMY_DATABASE_URI = str(os.getenv('SQLALCHEMY_DATABASE_URI', 'postgresql://navitia:navitia@localhost/chaos_cust'))
```
Comment (kun2985) [inline chaos/default_settings.py]: You may have uploaded the file by mistake

**Comment by kun2985** [comment]
May be you should create a feature branch and change the base of this pr to this feature branch ?

**Comment by pthegner** [comment]
There just one concern about this PR (very strange). For the file `chaos/models.py` that as not change since 2 month on the master branch, the lines that changes doesn't seems to be the same. Could you check it please ?

File: chaos/db_mapper/disruption.py [hunk 2]
```diff
@@ -181,7 +181,6 @@ def map_disruption(results):
                         'uri': r.line_section_end_uri,
                         'type': r.line_section_end_type
                     },
-                    'sens': r.line_section_sens,
                     'routes': [],
                     'via': [],
```
Comment (dvdn) [inline chaos/db_mapper/disruption.py:185]: Attribute 'via' is also useless

File: chaos/resources.py [hunk 3]
```diff
@@ -701,7 +701,6 @@ def post(self, contributor, navitia):
                             'uri': r.line_section_end_uri,
                             'type': r.line_section_end_type
                         },
-                        'sens': r.line_section_sens,
                         'routes': [],
                         'via': [],
```
Comment (dvdn) [inline chaos/resources.py:705]: Attribute 'via' is also useless
```suggestion
```

**Comment by tchevily** [comment]
Jenkins can you test this please?

**Pull request merged**